Doxyfile

```diff
@@ -79,7 +79,7 @@ LAYOUT_FILE            =
 
 # When adding extra files, separate them with spaces.
 
-INPUT                  = webgpu.h
+INPUT                  = webgpu.h doc/
```

Comment:
I'm a little iffy on this because `doc/` contains `doc/generated/`.

Also, the other day I figured out a way to get articles to group under a heading instead of next to `Topics` and `Data Structures` (not that this is too bad). In there I used `doc/articles` as the source dir.

https://github.com/webgpu-native/webgpu-headers/compare/main...kainino0x:webgpu-headers:articles